basket/news/views.py

```diff
@@ -228,6 +228,7 @@ def fxa_callback(request):
 @require_POST
 @csrf_exempt
 def confirm(request, token):
+    token = str(token)
```

Comment:
These changes are due to the [`<uuid:token>` URL components](https://docs.djangoproject.com/en/2.2/topics/http/urls/#path-converters) returning [`UUID` objects](https://docs.python.org/3/library/uuid.html#uuid.UUID) instead of strings.